pkg/icingadb/db.go

```diff
@@ -54,6 +54,10 @@ type Options struct {
 	// MaxRowsPerTransaction defines the maximum number of rows per transaction.
 	// The default is 2^13, which in our tests showed the best performance in terms of execution time and parallelism.
 	MaxRowsPerTransaction int `yaml:"max_rows_per_transaction" default:"8192"`
+
+	// WsrepSyncWait defines which kinds of SQL statements catch up all pending sync between nodes first, see:
+	// https://mariadb.com/kb/en/galera-cluster-system-variables/#wsrep_sync_wait
+	WsrepSyncWait int `yaml:"wsrep_sync_wait" default:"7"`
```

Comment:
This should also be documented in a place that users can easily find. The current example config does not contain an `options` block. This could perhaps be added, as for many people a look at the default config is a starting point.

Textual documentation would also be helpful. Perhaps also referring to the HA area?

Reply:
> The current example config does not contain an `options` block.

That's exactly my reason for not documenting it (yet).

Reply:
The reasoning for the existing options was that those those should not need to be changed by users. The default values should work fine everywhere, they were merely exposed to the config so that we might try to tweak them even in some production environment if problems would arise (but so far, those defaults seem to have worked out fine).